pkg/controllers/managementagent/servicemonitor/utils.go

```diff
@@ -77,6 +77,7 @@ func getServiceMonitorFromWorkload(w *util.Workload) (*monitoringv1.ServiceMonit
 		},
 		Spec: monitoringv1.ServiceMonitorSpec{},
 	}
+	_ = workloadTargetAnnotation
```

Comment:
Same question as above, I don't see this in the output

Reply:
`ineffassign`